schemas/skills/samples/complex-skillmanifest-2.1.preview-0.json

```diff
@@ -0,0 +1,184 @@
+{
+    "$schema": "https://schemas.botframework.com/schemas/skills/skill-manifest-2.1.preview-0.json",
+    "$id": "SkillBot",
+    "name": "Sample skill definition that can handle multiple types of activities",
+    "version": "1.0",
+    "description": "This is a sample skill definition for multiple activity types",
+    "publisherName": "Microsoft",
+    "privacyUrl": "https://myskill.contoso.com/privacy.html",
+    "copyright": "Copyright (c) Microsoft Corporation. All rights reserved.",
+    "license": "",
+    "iconUrl": "https://myskill.contoso.com/icon.png",
+    "tags": [
+        "sample",
+        "travel",
+        "weather"
+    ],
+    "endpoints": [
+        {
+            "name": "americas",
+            "protocol": "BotFrameworkV3",
+            "description": "Production  endpoint for SkillBot in the Americas",
+            "endpointUrl": "http://myskill.contoso.com/api/messages",
+            "msAppId": "00000000-0000-0000-0000-000000000000"
+        },
+        {
+            "name": "eu",
+            "protocol": "BotFrameworkV3",
+            "description": "Production  endpoint for SkillBot in Europe",
+            "endpointUrl": "http://myskill.contoso.com/api/messages",
+            "msAppId": "11111111-0000-0000-0000-000000000000"
+        }
+    ],
+    "dispatchModels": {
+        "languages": {
+            "en": [
+                {
+                    "id": "SkillLuModel-en",
+                    "name": "SkillBot LU (English)",
+                    "contentType": "application/lu",
```

Comment:
where are these types defined?

Reply:
We don't have constants for it now, we are leaving it open for now, sort of like with http requests content types where you have well known ones but you can put whatever you want.
Once we build a manifest parser we may be more strict but for now, we are assuming everything is lu files so the only one I know off is application/lu.